deltacat/compute/compactor/model/round_completion_info.py

```diff
@@ -6,6 +6,22 @@
 from typing import Any, Dict, Optional, Union
 
 
+class HighWaterMark(dict):
```

Comment:
Minor: Watermark is one word.

```suggestion
class HighWatermark(dict):
```